editor/action_map_editor.cpp

```diff
@@ -429,6 +429,7 @@ void ActionMapEditor::update_action_list(const Vector<ActionInfo> &p_action_info
 		// Update Tree...
 
 		TreeItem *action_item = action_tree->create_item(root);
+		ERR_FAIL_NULL(action_item);
```

Comment:
This should never trigger, added just in case.

Reply:
I know this comes from the issue, but there are more potentially crashing spots like that. Kinda odd to handle only this one specifically.